www/includes/easyparliament/templates/html/static/help.php

```diff
@@ -4,264 +4,396 @@
             <h1>Frequently Asked Questions</h1>
 
             <ul>
-            <li><a href="#use">How do people use TheyWorkForYou?</a></li>
-            <li><a href="#coverage">Is this the whole of Hansard?</a></li>
-            <li><a href="#moderation">How do you moderate annotations?</a></li>
-            <li><a href="#privacy">What is your privacy policy?</a></li>
-            <li><a href="#cookie">What is your cookie policy?</a></li>
-            <li><a href="#survey">What are these survey questions?</a></li>
-            <li><a href="#extreme">What's that weird 'extreme tracker' and 'urchin' javascript I spy in your page source?</a></li>
-            <li><a href="#rss">What is RSS?</a></li>
-            <li><a href="#votingrecord">How is the voting record decided?</a></li>
-            <li><a href="#numbers">Why should I read in more depth than just the numbers?</a></li>
-            <li><a href="#api">Do you have the data as a spreadsheet file, XML or in an API?</a></li>
+                <li><a href="#first">First things first</a></li>
+                <li><a href="#content">Content</a></li>
+                <li><a href="#data">Data and code</a></li>
+                <li><a href="#privacy">Privacy</a></li>
+                <li><a href="#problems">Problems</a></li>
             </ul>
 
-            <br>
+            <h2 id="first">First things first</h2>
+            <dl>
+                <dt id="what">
+                    What is TheyWorkForYou?
+                </dt>
+                <dd>
+                    TheyWorkForYou is a website with a simple aim: to make it much easier for anyone to understand exactly what is going on in Parliament.
+                </dd>
+
+                <dt id="government">
+                    Is it run by the government?
+                </dt>
+                <dd>
+                    No: it&rsquo;s a completely independent project, which has been run by the UK charity <a href="https://www.mysociety.org/">mySociety</a> since 2004.
```

Comment:
Reading mySociety's history page ;) it only became a mySociety project in April 2006.